example/fine_advection/advection_package.hpp

```diff
@@ -33,14 +33,23 @@ namespace advection_package {
 using namespace parthenon::package::prelude;
 
 namespace Conserved {
-VARIABLE(advection, scalar);
-VARIABLE(advection, scalar_fine);
-VARIABLE(advection, scalar_fine_restricted);
+VARIABLE(advection, phi);
```

Comment:
We are using the `VARIABLE` macro in various places throughout Parthenon now.  Not for this MR, but should we centrally locate this macro so that even downstream applications can use it?

Reply:
yeah, I think this makes sense, but maybe for another PR.